src/main/java/org/folio/rest/impl/OverdueFinePoliciesAPI.java

```diff
@@ -48,6 +54,12 @@ public void postOverdueFinesPolicies(String lang,
     Handler<AsyncResult<Response>> asyncResultHandler,
     Context vertxContext) {
 
+    if (isAnyNegativeQuantity(entity)) {
```

Comment:
Validate via json-schema

Reply:
It is a custom class and a validation it would be pretty hard. Let's stay with current.